lib/kernel/src/trace.erl

```diff
@@ -1007,10 +1000,6 @@ Set trace pattern for _message receiving_.
 Must be combined with `process/4` or `port/4` to set the `'receive'` trace flag
 for one or more processes or ports.
 
-By default all messages received by `'receive'` traced processes are
-traced. Use `trace:recv/3` to limit traced receive events based on the message
-content, the sender and/or the receiver.
-
```

Comment:
Same here.

Reply:
I misunderstood. I will add it back as soon as RC3 has been released.